Misc/NEWS.d/next/Build/2019-11-30-21-53-38.bpo-28833.14IeXH.rst

```diff
@@ -0,0 +1,9 @@
+Fix cross-compilation of third-party extension modules. The
```

Comment:
This change is backward incompatible. It breaks build systems relying on _PYTHON_xxx environment variables for cross-compilation. It should be documented somewhere near https://docs.python.org/dev/whatsnew/3.9.html#porting-to-python-3-9